### DISCUSSION TITLE
Haven't sent any `ref` to <Text /> but receiving this error about `forwardRef()` missing

### DISCUSSION THREAD

**Discussion by SBagaria2710** [zeit-ui/react#741] [Q&A]
<img width="710" alt="Screenshot 2022-02-13 at 1 17 33 PM" src="https://user-images.githubusercontent.com/33802361/153745836-443a968c-cb98-4708-b4ff-2626e09903ed.png">

**Comment by unix** [comment]
Yes, there are only a few form components (e.g. `Input` `Button`) that export `ref` externally, and we recommend always using the controlled component.
